src/utils/date.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Converts a date object into a string of the format yyyymmddHHMMss
+ * @param date A Date object
+ * @returns {string} String of the format yyyymmddHHMMss
+ */
+function formatDate(date: Date): string {
```

Comment:
so proposal about additional argument has been discarded? (since the file has been moved discussion has been lost)

Reply:
In the previous discussion these were my thoughts about this:

I don't remember a use for for different formats in formatDate in the original codebase. If the need arises that we need an option for different formats we can make the change then (it will be an easy change). I feel like we shouldn't introduce unnecessary dependencies if we don't even end up using different formats.